client/me/help/help-contact-form/index.jsx

```diff
@@ -233,6 +238,11 @@ module.exports = React.createClass( {
 				<FormLabel>{ this.translate( 'What are you trying to do?' ) }</FormLabel>
 				<FormTextarea valueLink={ this.linkState( 'message' ) } placeholder={ this.translate( 'Please be descriptive' ) }></FormTextarea>
 
+				{ showHelpLanguagePrompt && (
+					<div className="help-contact-form__help-language-prompt">
+						<FormLabel>{ this.translate( 'Note: The response will be in English.' ) }</FormLabel>
```

Comment:
I don't think we need `FormLabel` here. Maybe we can just add the styles you need to `.help-contact-form__help-language-prompt`